automation/emit_coverage_info.sh

```diff
@@ -41,7 +41,7 @@ export RUSTDOCFLAGS="-Cpanic=abort"
 # Apparently --no-default-features doesn't work in the root, even with -p to select a specific package.
 # Instead we pull the list of individual package manifest files which have default features
 # out of `cargo metadata` and test using --manifest-path for each individual package.
-for manifest in $(cargo metadata --no-deps --format-version 1 | jq -r '.packages[] | select((.features | .default | length) > 0) | .manifest_path'); do
+for manifest in $(cargo metadata --no-deps --format-version 1 | jq -r '.packages[] | select((.features | .default | length) > 0 or .name == "nimbus-fml") | .manifest_path'); do
```

Comment:
This is one thing I want to make sure we're okay with — since we're using the FML code in a production environment now I felt it was worth adding its code coverage to what's checked in CI. Seems it's existing coverage is pretty great!

Reply:
I think I'm okay with this. @bendk may have thoughts.

Reply:
I have 0 thoughts on this one :).  I don't understand how the code coverage works in general.